### DISCUSSION TITLE
JSON Report: Json schema or Go struct?

### DISCUSSION THREAD

**Discussion by JWebDev** [kubescape/kubescape#1715] [Q&A]
Hi,

I can't find in the documentation/repository the structure/schema of a json report or at least the data structure/template used to generate json report.
I can of course open the report and understand the structure, but it would be better to see the actual schema.

Thanks!

**Comment by matthyx** [comment]
cc @Oshratn

**Reply by JWebDev** [reply]
Hi, @matthyx @Oshratn  - any updates? It will be very helpful. Thanks.

**Reply by Bezbran** [reply]
Hi @JWebDev, [here](https://github.com/kubescape/opa-utils/blob/343b8c35d55ea0cdbbca45d287c02459406246b3/reporthandling/v2/datastructures.go#L16) is the go struct of the report.
Can you elaborate regarding your needs in it? Maybe we can assist you along the way.

**Reply by JWebDev** [reply]
@Bezbran Thank you so much for your wish to help. Do not close this ticket please. I will revisit this issue over the next few weeks.

**Reply by Bezbran** [reply]
You welcome
